src/System.Net.Security/tests/FunctionalTests/SslStreamSystemDefaultsTest.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Test.Common;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Authentication;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    using Configuration = System.Net.Test.Common.Configuration;
+
+    public class SslStreamSystemDefaultTest
+    {
+        [Fact]
+        public async Task SslStream_DefaultTlsConfigurationSync_Ok()
+        {
+            using (var test = new SyncTest())
+            {
+                await test.RunTest();
+            }
+        }
+
+        [Fact]
+        public async Task SslStream_DefaultTlsConfigurationApm_Ok()
+        {
+            using (var test = new ApmTest())
+            {
+                await test.RunTest();
+            }
+        }
+
+        [Fact]
+        public async Task SslStream_DefaultTlsConfigurationAsync_Ok()
+        {
+            using (var test = new AsyncTest())
+            {
+                await test.RunTest();
+            }
+        }
+
+        public abstract class TestBase : IDisposable
+        {
+            protected SslStream _clientStream;
+            protected SslStream _serverStream;
+
+            public TestBase()
+            {
+                var network = new VirtualNetwork();
+                var clientNet = new VirtualNetworkStream(network, false);
+                var serverNet = new VirtualNetworkStream(network, true);
+
+                _clientStream = new SslStream(clientNet, false, AllowAnyServerCertificate);
+                _serverStream = new SslStream(serverNet, false, AllowAnyServerCertificate);
+            }
+
+            public async Task RunTest()
+            {
+                X509Certificate2 serverCertificate = Configuration.Certificates.GetServerCertificate();
+                string serverHost = serverCertificate.GetNameInfo(X509NameType.SimpleName, false);
+                X509CertificateCollection clientCertificates = new X509CertificateCollection();
+                clientCertificates.Add(Configuration.Certificates.GetClientCertificate());
+
+                var tasks = new Task[2];
+                tasks[0] = AuthenticateClient(serverHost, clientCertificates, checkCertificateRevocation: false);
+                tasks[1] = AuthenticateServer(serverCertificate, clientCertificateRequired:true, checkCertificateRevocation:false);
+                await Task.WhenAll(tasks);
+
+                
+                if (PlatformDetection.IsWindows && PlatformDetection.WindowsVersion > 10)
```

Comment:
Greater than 10?  Did I miss an announcement? :smile:  Or does this mean the AU or CU versions?